### PR TITLE
doc improvement: Doc fixes for 2.4

### PR DESCRIPTION
**Pull request by umapraseeda**
Doc fixes for 2.4

### CODE REVIEW

File: doc/nrf/app_dev/bootloaders_and_dfu/bootloader_adding.rst [hunk 1]
```diff
@@ -342,5 +342,5 @@ This is a necessary step for creating application update images for use with :re
 The S1 variant is built as a separate child image called ``s1_image``.
 For this reason, any modifications to the configuration of the S1 variant must be done to the ``s1_image`` child image.
 By default, this child image is an exact duplicate of the original image, with the exception of its placement in memory.
-The only configuration option that must be modified is the version set in ``CONFIG_FW_INFO_FIRMWARE_VERSION``.
-To make ``s1_image`` bootable with |NSIB|, the value of ``CONFIG_FW_INFO_FIRMWARE_VERSION`` for ``s1_image`` must be bigger than the one for original image.
+The only configuration option that must be modified is the version set in :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION`.
```
Comment (peknis) [inline doc/nrf/app_dev/bootloaders_and_dfu/bootloader_adding.rst:345]:
```suggestion
You only have to modify the version set in the :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` option.
```

File: doc/nrf/app_dev/optimizing/power.rst [hunk 2]
```diff
@@ -101,7 +101,7 @@ Possible cases when PSM cannot be used:
 
 You can detect the first two cases by checking the ``+CEREG`` notification or ``AT%XMONITOR`` command output for Active-Time granted by the network.
 If the Active-Time is ``11100000``, the timer is deactivated and PSM cannot be used.
-Using the :ref:`lte_lc_readme` library, the Active-Time is notified with the :c:enum:`LTE_LC_EVT_PSM_UPDATE` event and can be queried using the :c:func:`lte_lc_psm_get` function.
+Using the :ref:`lte_lc_readme` library, the Active-Time is notified with the :c:member:`LTE_LC_EVT_PSM_UPDATE` event and can be queried using the :c:func:`lte_lc_psm_get` function.
```
Comment (peknis) [inline doc/nrf/app_dev/optimizing/power.rst:104]: Again, not so sure about the changed tagging. The current one works fine.

File: samples/tfm/tfm_psa_template/README.rst [hunk 3]
```diff
@@ -130,7 +130,7 @@ Generate security keys if needed:
     python3 bootloader/mcuboot/scripts/imgtool.py keygen -t ecdsa-p256 -k /home/user/ncs/_keys/mcuboot_priv.pem
     python3 bootloader/mcuboot/scripts/imgtool.py keygen -t ecdsa-p256 -k /home/user/ncs/_keys/nsib_priv.pem
 
-Update the ``child_image/mcuboot/prj.conf`` file to set the private signing key for MCUBoot:
+Update the ``child_image/mcuboot/prj.conf`` file to set the private signing key for MCUboot:
```
Comment (peknis) [inline samples/tfm/tfm_psa_template/README.rst:133]:
```suggestion
Update the :file:`child_image/mcuboot/prj.conf` file to set the private signing key for MCUboot:
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

File: doc/nrf/protocols/zigbee/configuring.rst [hunk 4]
```diff
@@ -198,8 +198,8 @@ To do this, configure the related Kconfig option for one or more modules:
 * :kconfig:option:`CONFIG_ZBOSS_TRACE_LOG_LEVEL`
 * :kconfig:option:`CONFIG_ZBOSS_OSIF_LOG_LEVEL`
 * :kconfig:option:`CONFIG_ZIGBEE_SHELL_LOG_LEVEL`
-* :kconfig:option:`CONFIG_ZIGBEE_APP_UTILS_LOG_LEVEL`
-* :kconfig:option:`CONFIG_ZIGBEE_LOGGER_EP_LOG_LEVEL`
+* :kconfig:option:`CONFIG_ZIGBEE_APP_UTILS_LOG_LEVEL_CHOICE`
+* :kconfig:option:`CONFIG_ZIGBEE_LOGGER_EP_LOG_LEVEL_CHOICE`
 * :kconfig:option:`CONFIG_ZIGBEE_SCENES_LOG_LEVEL`
```
Comment (MarekPieta) [inline doc/nrf/protocols/zigbee/configuring.rst:203]: Should we also update the remaining options and use `_LOG_LEVEL_CHOICE` suffix for them?

File: applications/nrf5340_audio/README.rst [hunk 5]
```diff
@@ -505,14 +505,14 @@ Selecting the BIS mode
 ======================
 
 The CIS mode is the default operating mode for the application.
-You can switch to the BIS mode by adding the :kconfig:option:`CONFIG_TRANSPORT_BIS` Kconfig option set to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
+To switch to the BIS mode, add and set the :ref:`CONFIG_TRANSPORT_BIS <config_nrf53_audio_app_options>` Kconfig option to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.
```
Comment (MarekPieta) [inline applications/nrf5340_audio/README.rst:508]: `To switch to the BIS mode, add and set the :ref:`CONFIG_TRANSPORT_BIS <config_nrf53_audio_app_options>` Kconfig option to ``y`` to the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.`
->
`To switch to the BIS mode, set the :ref:`CONFIG_TRANSPORT_BIS <config_nrf53_audio_app_options>` Kconfig option to ``y`` in the :file:`prj.conf` file for the debug version and the :file:`prj_release.conf` file for the release version.`

Comment (MarekPieta) [reply applications/nrf5340_audio/README.rst:508]: By adding a `CONFIG_TRANSPORT_BIS=y` to the prj.conf file we actually modify a value of a Kconfig option.

File: applications/nrf5340_audio/README.rst [hunk 6]
```diff
@@ -524,7 +524,7 @@ Selecting the CIS bidirectional communication
 =============================================
 
 The CIS unidirectional mode is the default operating mode for the application.
-You can switch to the bidirectional mode by adding the :kconfig:option:`CONFIG_STREAM_BIDIRECTIONAL` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
+To switch to the bidirectional mode, add and set the :ref:`CONFIG_STREAM_BIDIRECTIONAL <config_nrf53_audio_app_options>` Kconfig option to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
```
Comment (MarekPieta) [inline applications/nrf5340_audio/README.rst:527]: `add and set` -> `set`?

**Review by MarekPieta — APPROVED** [review]
A few small comments to consider. Apart from this looks good

File: applications/nrf5340_audio/README.rst [hunk 7]
```diff
@@ -533,7 +533,7 @@ Enabling the walkie-talkie demo
 
 The walkie-talkie demo uses one or two bidirectional streams from the gateway to one or two headsets.
 The PDM microphone is used as input on both the gateway and headset device.
-You can switch to using the walkie-talkie by adding the :kconfig:option:`CONFIG_WALKIE_TALKIE_DEMO` Kconfig option set to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
+To switch to using the walkie-talkie, add and set the :ref:`CONFIG_WALKIE_TALKIE_DEMO <config_nrf53_audio_app_options>` Kconfig option to ``y``  to the :file:`prj.conf` file (for the debug version) or to the :file:`prj_release.conf` file (for the release version).
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply applications/nrf5340_audio/README.rst]: Modified.

File: doc/nrf/protocols/zigbee/configuring.rst (hunk 4 above)
Comment (umapraseeda) [reply doc/nrf/protocols/zigbee/configuring.rst:203]: @wiba-nordic

**Review by wiba-nordic — APPROVED** [review]

File: doc/nrf/protocols/zigbee/configuring.rst (hunk 4 above)
Comment (sebastiandraus) [reply doc/nrf/protocols/zigbee/configuring.rst:203]: @MarekPieta , Why should we add this suffix? Also, CONFIG_ZIGBEE_APP_UTILS_LOG_LEVEL is not directly user-configurable. 
Zigbee usually uses `zephyr/subsys/logging/Kconfig.template.log_config` template. This template already has `choice "$(module)_LOG_LEVEL_CHOICE"`, adding another _CHOICE in module name seems weird.
How about modifying the template?

**Review by b-gent — APPROVED** [review]

**Pull request merged**